example_client/README.md

```diff
@@ -18,6 +18,11 @@ In the examples listed below, OVMS can be started using a command:
 docker run -d --rm -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path gs://ovms-public-eu/resnet50 --port 9000 --rest_port 8000
 ```
 
+Since access to www resources is needed for the samples to operate add proxy settings variables to the above command if required:
```

Comment:
I suggest avoiding duplicate command with proxy settings. It should be sufficient to add a note that access to Google Cloud Storage might require proper configuration of https_proxy in the docker engine or in the docker container. Or combine the commands in one with https_proxy env. `-e https_proxy=${https_proxy}` might be a bit more elegant.